config/store.go

```diff
@@ -302,12 +303,7 @@ func (s *Store) updateConfigTx(ctx context.Context, tx *sql.Tx, fn func(Config)
 		return 0, err
 	}
 
-	data, err := json.Marshal(newCfg)
-	if err != nil {
-		return 0, errors.Wrap(err, "marshal config")
-	}
-
-	data, err = mergeJSON(cfg.data, data)
```

Comment:
We should be able to safely delete this func now, too (currently only being referenced here and in a test) now that `Merge()` has been created

Reply:
Good catch, I think I mistakenly re-introduced these two files when juggling `git stash`-es.